paperqa/types.py

```diff
@@ -426,6 +432,16 @@ def inject_clean_doi_url_into_data(data: dict[str, Any]) -> dict[str, Any]:
 
         return data
 
+    @staticmethod
+    def remove_invalid_authors(data: dict[str, Any]) -> dict[str, Any]:
+        """Capture and cull strange author names."""
+        if authors := data.get("authors"):
+            data["authors"] = [
+                a for a in authors if a.lower() not in {"et al", "et al."}
```

Comment:
Can you move the `et al` to a module level or class variable with a descriptive name:

```python
INVALID_AUTHORS_NAMES = {"et al", "et al."}  # We exact match these
```